diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -965,7 +965,12 @@ int DifferentialFunctionComparator::cmpGlobalValues(GlobalValue *L,
                 }
             }
             return 0;
-        } else
+        } else if (NameL != NameR && GVarL && GVarR && !GVarL->hasInitializer()
+                   && !GVarR->hasInitializer()) {
```

Comment:
Also add a comment here explaining what happens. Something like "Externally defined constants (those without initializer) need to have their definitions linked."